.secrets.baseline

```diff
@@ -114,5 +114,5 @@
       }
     ]
   },
-  "generated_at": "2021-12-08T00:06:35Z"
+  "generated_at": "2021-12-08T19:38:46Z"
```

Comment:
No need to include this in the patch, I think

Reply:
I believe our approval settings are currently to [dismiss stale reviews](https://docs.github.com/en/repositories/configuring-branches-and-merges-in-your-repository/defining-the-mergeability-of-pull-requests/about-protected-branches#require-pull-request-reviews-before-merging) every time there's a change in the commit manifest of the PR